livy/src/main/resources/interpreter-setting.json

```diff
@@ -160,6 +160,17 @@
   },
   {
     "group": "livy",
+    "name": "pyspark3",
+    "className": "org.apache.zeppelin.livy.LivyPySpark3Interpreter",
+    "properties": {
+    },    
```

Comment:
should this have a `option` group like `"org.apache.zeppelin.livy.LivyPySparkInterpreter` above?